### PR TITLE
Revise the section of `linearizable ledger validation` in README

### PR DESCRIPTION
**Pull request by supl**
This PR revised README according to Jun's advice in https://github.com/scalar-labs/scalardl-javascript-sdk-base/pull/85#pullrequestreview-1105136846

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -120,20 +120,22 @@ const statusCode = response.getCode();
 const proof = response.getProof();
 ```
 
-#### Validate an asset linearizably
-The default ledger validation in a Auditor-enabled Scalar DL network is non-linearizable; i.e., there might be cases where Ledger and Auditor look inconsistent temporarily.
-Scalar DL supports linearizable ledger validation.
-To use it, we can configure the properties as follows
+#### Ledger validation when Auditor is used
```
Comment (feeblefakie) [inline README.md:123]: Why don't you refer to the following doc instead explaining the same?

https://github.com/scalar-labs/scalardl/blob/master/docs/getting-started-auditor.md#validate-the-states-of-ledger-and-auditora

Comment (supl) [reply README.md:123]: Sorry, my bad!
Thank you for the advice.
I revised it in https://github.com/scalar-labs/scalardl-node-client-sdk/pull/83/commits/cd231ee2a6d11129a6cbf5ef3f834b75917ffa2b

**Review by jnmt — COMMENTED** [review]
Sorry for the late review. I left a comment. PTAL! Thank you!

File: README.md (hunk 1 above)
Comment (supl) [reply README.md:123]: @jnmt 
Jun, Thank you for your advice.
I revised it in https://github.com/scalar-labs/scalardl-node-client-sdk/pull/83/commits/116eb48067a468f1ebf03912a7f580e576244b24

Please take a look.
Thank you!

**Review by jnmt — APPROVED** [review]
LGTM! Thank you for the fix!

**Review by feeblefakie — APPROVED** [review]
LGTM! Thank you!

**Pull request merged**